### DISCUSSION TITLE
How to deal with Cloudflare Queues with Hono

### DISCUSSION THREAD

**Discussion by Makisuo** [honojs/hono#1163] [Q&A] [answered]
Hi I have been using Hono for a while now and haven't had much problems yet, though I have run into the issue of wanting to consume a cloudflare queue in the same worker as my consumer using hono.
Now I was not sure what's the best way to implement a queue consumer in that context with honor, any pointers would be appreciated!

**Comment by yusukebe** [comment, marked as answer]
Just write this:

```ts
export default {
  fetch: app.fetch,
  async queue(...): {}
}
```

**Reply by Daniel-Ash** [reply]
Would be helpful to mention this in the docs as this was a point of confusion for me when I was evaluating whether to use Hono :)
